src/nxdoc/getting-started/best-practices/nuxeo-platform-feature-table.md

```diff
@@ -800,21 +800,21 @@ The tables below use the following notation:
     <td colspan="1">Default distribution</td>
     <td>N/A</td>
     <td>&#10003;</td>
-    <td>N/A</td>
+    <td>&#10003;</td>
     <td colspan="1">
-    [User documentation]({{page space='userdoc' page='document-publishing'}})<br/>
-    [Developer documentation]({{page page='publisher'}})
+    [User Documentation]({{page space='userdoc' page='document-publishing'}})<br/>
+    [Developer Documentation]({{page page='publisher'}})
     </td>
 </tr>
 <tr>
     <td colspan="1">Bulk Publication  </td>
     <td colspan="1">Default distribution</td>
     <td>N/A</td>
     <td>&#10003;</td>
```

Comment:
Afaik, JSF does not support Bulk Publication. 
Bulk publication should allow user to select several items and publish them in one action.
So, if correct, it should be Web UI only feature. (publish a forlder is another topic)